docs/websocket/feedback.md

```diff
@@ -63,14 +63,46 @@ the following json is sent as a progressive result:
 ```json
 {
   "sentence": 0,
-  "feedback": true
+  "feedback": true,
+  "words": [
+    {
+      "expected": "funny",
+      "recognized": "funny",
+      "error": null,
+      "correct": true
+    },
+    {
+      "expected": "gif",
+      "recognized": "jif",
+      "error": "substitution",
+      "correct": false
+    }
+  ]
 }
 ```
 
 Name     | Type   | Description
 ---------|--------|------------
 sentence | `int`  | The index of the sentence, starting at `0`.
 feedback | `bool` | Feedback score, `false` means bad and `true` means good.
+words    | `list` | The expected and recognized words in the sentence.
```

Comment:
At this point I assume that we will always return ALL words that were expected in a single sentence. If so, it might be good to note that as wel.

Reply:
Done